### DISCUSSION TITLE
Firebase analytics - Custom parameter not visible in firebase console

### DISCUSSION THREAD

**Discussion by andy-thestunt** [firebase/flutterfire#12689] [Q&A]
I have implemented firebase analytics in my flutter application using firebase_analytics plugin. I have also used custom parameters to with the events. But I am facing problem that I cannot see that parameters in events section along with the events. Can you please help me how can I show that parameters along with events. Please see attached as a reference.

<img width="1110" alt="screenshot_1" src="https://github.com/firebase/flutterfire/assets/146887178/c7e692de-50d5-42cb-9931-a41733ff44ab">

**Comment by JeffHyde** [comment]
I have the same issue.  The solution I found is not ideal...  If you click into an event it will navigate to the "Analytics Dashboard".  On the right side of the dash board you'll see "EVENTS IN LAST 30 MINUTES" this box has a drop down to select a parameter.   As of todays date you cannot change the time frame so you are stuck with the last 30 min.

**Comment by kirstenkoa** [comment]
I had the same issue. I had to add "Custom definitions" in the Firebase Analytics console Settings. After adding it, any _new_ events with those params will show up in the dashboard.

https://firebase.google.com/docs/analytics/user-properties?platform=flutter#set_user_properties_2
